### PR TITLE
Fixes to get piedpiper installer working on DO Ubuntu 18.04

### PR DESCRIPTION
**Pull request by dingus9**

### CODE REVIEW

**Comment by ragingpastry** [comment]
#nitpick commit messages!

https://chris.beams.io/posts/git-commit/

**Review by codrcodz — COMMENTED** [review]
Your .gitignore changes conflict with previous PR.

**Review by codrcodz — CHANGES_REQUESTED** [review]
Please resolve merge conflicts to .gitignore.

**Comment by dingus9** [comment]
Merge conflicts fixed, I did chose to leave the ansible driver in for now so that I can push configs... the pull method doesn't let me deploy from local changes. I will test the pull method again once these changes are merged in. Pull method pulls from master after all.

File: bootstrap.sh [hunk 1]
```diff
@@ -52,14 +51,11 @@ else
   exit 1;
 fi
 
-which pip2 || curl -s https://bootstrap.pypa.io/get-pip.py | python2.7;
-pip2 install -U pip;
-which pip3 || curl -s https://bootstrap.pypa.io/get-pip.py | python3.6;
```
Comment (codrcodz) [inline bootstrap.sh:57]: get-pip is now the canonical method for installing pip according to the PyPA and ensures the user is always installing pacakges with the latest version of pip.

File: bootstrap.sh [hunk 2]
```diff
@@ -36,10 +36,9 @@ case $2 in
     ;;
 esac
 
-# bootstrap pidev env
 if [[ ${devenv_name} =~ ubuntu1804/* ]]; then
   apt-get update -y \
-  && apt-get install -y curl git python-dev gcc python3-distutils;
+  && apt-get install -y python3-pip;
```
Comment (codrcodz) [inline bootstrap.sh]: use get-pip to perform initial install of pip, and pip itself to upgrade pip thereafter.

File: bootstrap.sh (hunk 1 above)
Comment (codrcodz) [reply bootstrap.sh:57]: Also, by removing this line, you would break the minishift env.

**Review by codrcodz — CHANGES_REQUESTED** [review]

**Pull request merged**